src/zenml/integrations/mlflow/model_deployers/mlflow_model_deployer.py

```diff
@@ -138,7 +134,7 @@ def get_active_model_deployer() -> "MLFlowModelDeployer":
                 "You can create a new stack with an MLflow model "
                 "deployer component or update your existing stack to add this "
                 "component, e.g.:\n\n"
-                "  'zenml model-deployer register mlflow --type=mlflow'\n"
+                "  'zenml model-deployer register mlflow --flavor=mlflow'\n"
```

Comment:
I guess we could replace mlflow with the `MLFLOW_MODEL_DEPLOYER_FLAVOR` constant here?

Reply:
Thanks! Changed it.